pkg/snmp/snmpparse/config_snmp.go

```diff
@@ -0,0 +1,102 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Should we leave config_snmp.go file in `cmd/agent/subcommands/snmp` instead of having it in `pkg/snmp` ?

I'm not sure we need to share this code (by having it in `pkg/snmp`) right now since it's only useful to snmp command right now.